rct_optimizations/src/rct_optimizations/validation/noise_qualification.cpp

```diff
@@ -20,8 +20,8 @@ Eigen::Quaterniond computeQuaternionMean(const std::vector<Eigen::Quaterniond>&
   * M = sum(w_i * q_i * q_i^T)    Eq. 12
   * q_bar = argmax(q^T * M * q)   Eq. 13
   *
-  * The solution of this maximization problem is well known. The average quaternion is
-  * the eigenvector of M corresponding to the maximum eigenvalue.
+  * "The solution of this maximization problem is well known. The average quaternion is
+  * the eigenvector of M corresponding to the maximum eigenvalue."
```

Comment:
Are these "editorial" kinds of quotes?

Reply:
That was a quote that I pulled directly from the paper. I probably should do a better job citing it, but I at least wanted to indicate that that particular comment was not my own